.github/workflows/cicd-main.yml

```
@@ -0,0 +1,30 @@
+name: CI/CD
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Repository 접근
+        uses: actions/checkout@v4
+
+      - name: JDK 21 셋팅
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'temurin'
+
+      - name: 이미지 빌드 및 푸쉬
+        env:
+          REDIS_PORT: ${{ secrets.PROD_REDIS_PORT }}
+          REDIS_HOST: ${{ secrets.PROD_REDIS_HOST }}
+          DB_URL: ${{ secrets.PROD_DB_URL }}
+          DB_USER: ${{ secrets.PROD_DB_USER }}
+          DB_PASSWORD: ${{ secrets.PROD_DB_PASSWORD }}
+        run: |
+          DOCKER_HUB_USERNAME=${{ secrets.DOCKERHUB_USERNAME }} DOCKER_HUB_PASSWORD=${{ secrets.DOCKERHUB_PASSWORD }} ${{ secrets.DOCKERHUB_REPOSITORY }} ./gradlew :piikii-bootstrap:bootBuildImage
```

`DOCKER_HUB_USERNAME=${{ secrets.DOCKERHUB_USERNAME }} DOCKER_HUB_PASSWORD=${{ secrets.DOCKERHUB_PASSWORD }} ${{ secrets.DOCKERHUB_REPOSITORY }}`
위 부분 위의 `env`에서 설정하면 여기서 빼도 될 것 같아요. 그럼 더 깔끔하게 작성 가능할 듯...?

ex) 아래처럼? 확인이 필요함.
```suggestion
          DB_PASSWORD: ${{ secrets.PROD_DB_PASSWORD }}
          DOCKER_HUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
          DOCKER_HUB_PASSWORD: ${{ secrets.DOCKERHUB_PASSWORD }}
          DOCKER_HUB_REPOSITORY: ${{ secrets.DOCKERHUB_REPOSITORY }}
        run: ./gradlew :piikii-bootstrap:bootBuildImage
```

---

이거 그 gradlew 실행 옵션 자체에 저렇게 환경변수를 넣어야하는거같은데 음.... 한번 이렇게해서 확인해볼게여

---

아 걍 될거같네요 수정했습니다~